uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Page/AbsenceType.php

```diff
@@ -16,6 +16,15 @@ class CRM_HRLeaveAndAbsences_Page_AbsenceType extends CRM_Core_Page_Basic {
   public function run() {
     CRM_Utils_System::setTitle(ts('Leave/Absence Types'));
     $this->absenceTypeService = new AbsenceTypeService();
+    $action = CRM_Utils_Request::retrieve('action', 'String', $this, FALSE, 'browse');
+    $this->assign('action', $action);
+
+    if ($action & CRM_Core_Action::BROWSE) {
+      $this->browse();
+    } else {
+      $this->loadResources();
```

Comment:
The resources are not needed for browsing? I would have thought the browse would use some JS scripts?

Reply:
The JS scripts were added [here](https://github.com/compucorp/civihr/pull/2865/files#diff-d77b9ef8ffe49db8d6d55fb08ac88bd2R44). Will refactor since i have moved the implementation away from listing page. ♻️